tools/assignmentPlacer.py

```diff
@@ -0,0 +1,28 @@
+import os, sys
+import shutil
+
+gradeable_path = '/var/local/submitty/courses/s20/sample/submissions/grades_released_homework_autohiddenEC'
+
+if __name__ == "__main__":
+    if not os.path.isdir('assignments') or not os.path.isdir(gradeable_path):
+        print('The assignments directory or the gradeable does not exist. Please create these.')
+        sys.exit(1)
```

Comment:
`raise SystemExit('The assignments directory or the gradeable does not exist. Please create those.')`